src/dev/flang/parser/Parser.java

```diff
@@ -1757,45 +1726,9 @@ Actual actualSpace()
          ;
```

Comment:
grammar rule above seems outdated?

Reply:
Maybe I will change this, left it intentionally since it the OperatorExpr will be turned into a type. Have to think about this.